openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx

```diff
@@ -39,8 +39,9 @@ const FilterContainer: FunctionComponent<FilterContainerProp> = ({
 
   return (
     <div
-      className="filter-group tw-justify-between tw-mb-2"
-      data-testid={`filter-container-${name}`}>
+      className="filter-group tw-justify-between tw-mb-2 tw-cursor-pointer"
+      data-testid={`filter-container-${name}`}
+      onClick={() => onSelect(!isSelected, name, type)}>
```

Comment:
If we are adding it here, remove eventHandler from below, i.e. checkbox.
This will call it twice with same args